plankton/scripts/run.py

```diff
@@ -61,23 +62,23 @@
                     help='Arguments for the adapter.')
 
 
-def run_simulation(argument_list=None):
+def do_run_simulation(argument_list=None):
     arguments = parser.parse_args(argument_list or sys.argv[1:])
 
     if arguments.version:
-        print(__version__)
-        return
+        return __version__
```

Comment:
The advantage of returning is that we can have tests for the function and can check what it will actually print. Catching stdout just complicates the tests.